src/web.rs

```diff
@@ -280,5 +280,11 @@ where
     I: Send + 'static,
     E: Send + std::fmt::Debug + 'static,
 {
-    actix_threadpool::run(f).await
+    // map error to BlockingError. this is for not breaking the existing api.
+    // preferably this should just return actix_rt::task::JoinHandle for a more
+    // flexible control of when the task is awaited and/or the abort(if needed).
```

Comment:
we should make an issue to discuss this

Reply:
After a check from the tokio doc. It seems there is no canceling of blocking tasks. So I could be wrong on the abort of blocking task.

Reply:
I'm removing this comment and keep the code as it is for now. A unabortable task handle is not of much use.